cryptofeed/exchange/binance.py

```diff
@@ -40,9 +40,15 @@ def _parse_symbol_data(cls, data: dict, symbol_separator: str) -> Tuple[Dict, Di
             split = len(symbol['baseAsset'])
             normalized = symbol['symbol'][:split] + symbol_separator + symbol['symbol'][split:]
             ret[normalized] = symbol['symbol']
+
+            # Add subscribeable symbol for an index
+            index = '.' + symbol['pair'][:split] + symbol_separator + symbol['pair'][split:]
+            ret[index] = '.' + symbol['pair']
```

Comment:
We also need to add the dot to the exchange symbol, because otherwise these symbols would overlap with some of the Binance Delivery symbols (the dot will need to be stripped out in functions that actually integrate with the exchange which is not ideal).